.github/workflows/run-tests.yaml

```diff
@@ -0,0 +1,17 @@
+name: Run tests
+
+on: [push, pull_request]
+
+jobs:
+    test:
+        runs-on: ubuntu-latest
+        steps:
+            - uses: actions/checkout@v3
+            - name: Set up Python
+              uses: actions/setup-python@v4
+              with:
+                python-version: "3.10"
```

Comment:
lets go with 3.11 ;)
> Python 3.11 is between 10-60% faster than Python 3.10. On average, we measured a 1.25x speedup on the standard benchmark suite.

https://docs.python.org/3/whatsnew/3.11.html

Reply:
Well, as you can imagine all I did was copying the job from other repo, but I can change that for sure.

Reply:
Not done - 3.11 asyncio does not have @asyncio.coroutine decorator, and asynctest lib dies. We should drop the lib then, but I'll do it later. Here's the way to go https://fastapi.tiangolo.com/advanced/async-tests/